Rakefile

```diff
@@ -6,17 +6,21 @@ def init_api
   @init_api ||= OnlyofficeDigitaloceanWrapper::DigitalOceanWrapper.new
 end
 
+def init_digital_ocean_helper
```

Comment:
I think better name is `def digital_ocean_helper`

Since init is only performed one time on first call, and on other there is no init